wiki/Manual.md

```diff
@@ -2423,32 +2423,9 @@ If `since` is not specified the `fetchOHLCV` method will return the time range a
 
 ### Notes On Latency
 
-Trading strategies require fresh up-to-date information for technical analysis, indicators and signals. Building a speculative trading strategy based on the OHLCV candles received from the exchange may have critical drawbacks. Developers should account for the details explained in this section to build successful bots.
+All data that you are getting with CCXT is received directly from the exchanges' API engine. Typically, exchanges construct "secondary data" (OHLCV bars, tickers, etc) from the realtime "first-hand" data" (trades, orderbook ...). However, some exchanges might have imperfect engine, and their constructed OHLCV bars might have some imperfections too, for example, values not being calculated at :00 offset on universal clock, but few milli/seconds later, because it takes some time for the exchange hardware to collect the first-hand data and generate the secondary statistical data from it. So, there is always some latency between the moment when a trade happens and the moment when a corresponding OHLCV candle is updated & published by the exchange API. Some exchanges might do it in regular intervals like once a second or once in a few seconds, but some slow exchanges might take minutes to update the secondary statistical information.
 
-First and foremost, when using CCXT you're talking to the exchanges directly. CCXT is not a server, nor a service, it's a software library. All data that you are getting with CCXT is received directly from the exchanges first-hand.
```

Comment:
either here, it's just a ccxt definition, no place for those right in this part. otherwise, we should repeat same under every chapter